source/index.html.erb

```diff
@@ -1,32 +1,29 @@
 ---
 title: Index Page Example
 ---
-
 <div class="masthead">
   <div class="hero hero--breaded">
     <div class="hero__content">
       <div class="hero__body">
-        <h1 class="hero__title"> Join the single Wifi authentication service </h1>
-          <div class="hero__inline-image">
-            <img src="//placehold.it/646x492/005ea5/ffffff?text=A+supporting+image" alt="" role="presentation">
-          </div>
-          <p class="hero__description">
-            Install GovWifi so your users can stay connected in any building.
-          </p>
-          <%= link_to "Get GovWifi for your organisation", "https://admin.wifi.service.gov.uk/users/sign_up", class: "hero-button" %>
-          <p class="hero__user-text">
-            Check if you're ready for GovWifi in our <%= link_to "technical documentation", 'https://docs.wifi.service.gov.uk/requirements.html#requirements' %>.
-          </p>
-
-          <p class="hero__user-text">
-            Connecting to GovWifi as a user? See <%= link_to "instructions on how to connect", '/connect.html' %>.
-          </p>
+        <h1 class="hero__title">Join the single Wifi authentication service</h1>
+        <div class="hero__inline-image">
+          <img src="/images/Product-illustration-150-crop-2.png" alt="" role="presentation" width="100%">
+        </div>
+        <p class="hero__description">
+          Install GovWifi so your users can stay connected in any building.
+        </p>
+        <a href="https://admin.wifi.service.gov.uk/users/sign_up" class="hero-button">Get GovWifi for your organisation</a>
```

Comment:
I'd probably maintain the old link helpers for these - have a look at [link_to](https://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to), you can use it in erb templates like this.